### PR TITLE
Contact information app

### PR DESCRIPTION
**Pull request by stephendawsondev**
Adds the contact information form.

Right now, it's just one big form, so it still needs to be styled.

### CODE REVIEW

File: profiles/views.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
-from django.views.generic import TemplateView, CreateView, UpdateView
-from django.contrib.auth.mixins import UserPassesTestMixin, LoginRequiredMixin
```
Comment (Dee-McG) [inline profiles/views.py:2]: LoginRequiredMixin is needed & UserPassesTestMixin will be needed shortly

File: profiles/views.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
-from django.views.generic import TemplateView, CreateView, UpdateView
```
Comment (Dee-McG) [inline profiles/views.py:1]: You seem to of deleted the views being used in the other functions

**Review by Dee-McG — CHANGES_REQUESTED** [review]

File: profiles/views.py [hunk 3]
```diff
@@ -29,8 +37,30 @@ class UpdateSummary(LoginRequiredMixin, UpdateView):
...
-        return HttpResponse('<p class="success">Form submitted successfully!</p>')  
+        return HttpResponse('<p class="success">Form submitted successfully!</p>')
 
     def form_invalid(self, form):
         return HttpResponse('<p class="error">Please provide a summary. Max 500 chars</p>')
-  
\ No newline at end of file
+  
+
+class CreateUpdateContactInformation(LoginRequiredMixin, View):
```
Comment (Dee-McG) [inline profiles/views.py:50]: View is not imported

**Review by Dee-McG — CHANGES_REQUESTED** [review]

File: profiles/views.py (hunk 3 above)
Comment (stephendawsondev) [reply profiles/views.py:50]: Reimported in fix.

**Review by Dee-McG — APPROVED** [review]

**Pull request merged**